src/Compilers/Core/Portable/Compilation/IExpression.cs

```diff
@@ -795,31 +795,21 @@ public interface IObjectCreationExpression : IExpression
         /// <summary>
         /// Explicitly-specified member initializers.
         /// </summary>
-        ImmutableArray<IMemberInitializer> MemberInitializers { get; }
+        ImmutableArray<ISymbolInitializer> MemberInitializers { get; }
     }
 
     /// <summary>
     /// Represents an object member initializer.
     /// </summary>
-    public interface IMemberInitializer : IOperation
+    public interface ISymbolInitializer : IOperation
```

Comment:
It just make the implementations a bit simpler. The sharing is probably not of interest to consumers.